plothist/hep_plotters.py

```diff
@@ -141,21 +145,47 @@ def compare_data_mc(
 
     # Ignore divide-by-zero warning
     np.seterr(divide="ignore", invalid="ignore")
-    # Compute data/MC ratio
-    ratio = np.where(
-        mc_hist_total.values() != 0, data_hist.values() / mc_hist_total.values(), np.nan
-    )
-    # Compute scaled uncertainties
-    scaled_data_uncertainty = np.sqrt(data_hist.variances()) / data_hist.values()
-    scaled_mc_uncertainty = np.sqrt(mc_hist_total.variances()) / mc_hist_total.values()
+    if ratio == "diff":
+        # Compute data/MC ratio
+        ratio_values = np.where(
+            mc_hist_total.values() != 0,
+            data_hist.values() / mc_hist_total.values(),
+            np.nan,
+        )
+        # Compute scaled uncertainties
+        scaled_data_uncertainty = np.sqrt(data_hist.variances()) / data_hist.values()
+        scaled_mc_uncertainty = (
+            np.sqrt(mc_hist_total.variances()) / mc_hist_total.values()
+        )
+    elif ratio == "pull":
+        # Compute pulls
+        ratio_values = np.where(
+            data_hist.values() != 0,
+            (mc_hist_total.values() - data_hist.values())
```

Comment:
We should have a reference for the definition we use. I think that usually, it is data-mc, not the opposite